spec/lib/workos/user_management_spec.rb

```diff
@@ -306,4 +306,27 @@
       end
     end
   end
+  describe '.delete_user' do
+    context 'with a valid id' do
+      it 'returns true' do
+        VCR.use_cassette('user_management/delete_user_with_valid_id') do
```

Comment:
I started naming these into folders for cleanliness. 

`user_management/delete_user/valid` for example.

Reply:
I just changed the file structure to match this design pattern in this PR and the other PRs